config/default/webhook/manifests.yaml

```diff
@@ -31,10 +31,10 @@ webhooks:
         path: /mutate-pods
     failurePolicy: Fail
     name: inferenceservice.kfserving-webhook-server.pod-mutator
-    namespaceSelector:
+    objectSelector:
       matchExpressions:
-        - key: control-plane
-          operator: DoesNotExist
+        - key: serving.kubeflow.org/inferenceservice
+          operator: Exists
```

Comment:
Does this mean every InferenceService must have this label? If so would we not need to update all our examples?
Also, is this the keyt breaking change for 1.15 k8s? Note kubeflow have 1.14 support for 1.0 but maybe this will be dropped by the time this version of KFServing gets in Kubeflow I assume. https://www.kubeflow.org/docs/started/k8s/overview/#minimum-system-requirements

Reply:
@cliveseldon the inference service already has [this label](https://github.com/kubeflow/kfserving/blob/ff54d08749163fc7603e231edcc9dc6019a05f73/pkg/controller/inferenceservice/resources/knative/service.go#L166), we internally already use this with k8s 1.15. We did not add it because currently kubeflow min k8s requirement is 1.14 which does not support object selector, kubeflow is now working on to support 1.16, we can integrate KFServing once kubeflow moves to 1.15 or 1.16. Note that latest Knative min requirement is already [k8s 1.16](https://github.com/knative/serving/releases/tag/v0.15.1)

Reply:
ok yes. I forgot our webhook is for Pods not InferenceServices